mne/datasets/utils.py

```diff
@@ -251,7 +251,7 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
     path = _get_path(path, key, name)
     # To update the testing or misc dataset, push commits, then make a new
     # release on GitHub. Then update the "releases" variable:
-    releases = dict(testing='0.116', misc='0.8')
+    releases = dict(testing='0.117', misc='0.8')
```

Comment:
you also need to update the hash below

Reply:
Oh I thought I had already updated the release. Thanks